tracer/test/benchmarks/Benchmarks.Trace/Log4netBenchmark.cs

```diff
@@ -11,14 +11,17 @@
 namespace Benchmarks.Trace
 {
     [MemoryDiagnoser]
+    [InProcess]
```

Comment:
This attribute is breaking the benchmark comparison tool (which we can update if necessary), but I don't think it's necessary here?
```suggestion
```

Reply:
This was required to run locally (thanks new antivirus software). I'll remove this so it's not checked-in. Done in aff538a61d2115e82b5c3112ba5760cdb491234d